lib/build/bundle.js

```diff
@@ -194,6 +195,30 @@ exports.Bundle = class {
     });
   }
 
+  copyFiles() {
+    if (!this.config.dependencies) {
+      return Promise.resolve();
+    }
+
+    const output = this.bundler.project.platform.output;
+    const root = this.bundler.project.paths.root;
+    let filesToCopyList = this.config.dependencies
+      .filter(x => x instanceof Object && x[copyProperty]);
```

Comment:
Well, I guess a better question is why can't you just say `.filter(x => x.copy)`? That will return truthy if `copy` exists, and falsy otherwise, which should take care of your filtering.